project/BuildHelper.scala

```diff
@@ -109,8 +110,9 @@ object BuildHelper {
 
   def extraOptions(scalaVersion: String, optimize: Boolean) =
     CrossVersion.partialVersion(scalaVersion) match {
-      case Some((3, 0))  =>
+      case Some((3, _))  =>
         Seq(
+          "-explain-types",
```

Comment:
I don't know if this option is needed or not but because `scalacOptions` for scala2 already have `-explaintypes`  (defined in `val std2xOptions`) and the corresponding option for scala3 is `-explain-types` - so I add this for scala3 too.